modules/security-context-constraints-rbac.adoc

```diff
@@ -10,6 +10,11 @@ you to scope access to your SCCs to a certain project or to the entire
 cluster. Assigning users, groups, or service accounts directly to an
 SCC retains cluster-wide scope.
 
+[NOTE]
+====
+No SCC is assigned to pods created in one of the default namespaces (`default`, `kube-system`, `kube-public`, `openshift-node`, `openshift-infra`, `openshift`). These namespaces should not be used for running pods or services.
```

Comment:
```suggestion
No SCC is assigned to pods created in one of the default namespaces, `default`, `kube-system`, `kube-public`, `openshift-node`, `openshift-infra`, `openshift`. These namespaces should not be used for running pods or services.
```

Reply:
_IBM Style Guide_, "Parentheses in running text": Avoid parentheses in running text..

Reply:
_No SCC is assigned to pods created in one of the default namespaces:_ ?

Reply:
I don't know enough about SCCs. Should this sentence be active? 
_You cannot assign a SCC to a pod in a default namespace_?

Reply:
> Use a colon instead of the parens? _one of the default namespaces:_ ?
Thanks. I've updated the suggested change.